lib/index.js

```diff
@@ -0,0 +1,68 @@
+import { buildProcessPipeline } from "./util.js";
+import { readFile } from "node:fs/promises";
+import * as path from "node:path";
```

Comment:
Minor detail (same in `util.js`):
```suggestion
import path from "node:path";
```

Reply:
:heavy_check_mark: